setup.py

```diff
@@ -40,6 +40,94 @@
     removable = []
     extension = ""
 
+removable_folders = [
+    "lib/apt",
+    "lib/asn1crypto",
+    "lib/asyncio",
+    "lib/notebook",
+    "lib/xarray",
+    "lib/concurrent",
+    "lib/bokeh",
+    "lib/bs4",
+    "lib/certifi",
+    "lib/chardet",
+    "lib/click",
+    "lib/cloudpickle",
+    "lib/cryptography",
+    "lib/curses",
+    "lib/cftime",
+    "lib/dask",
+    "lib/dateutil",
+    "lib/distributed",
+    "lib/docutils",
+    "lib/fabio",
+    "lib/gi",
+    "lib/idna",
+    "lib/ipykernel",
+    "lib/IPython",
+    "lib/ipython_genutils",
+    "lib/ipywidgets",
+    "lib/jinja2",
+    "lib/jupyter_client",
+    "lib/jupyter_core",
+    "lib/lib2to3",
+    "lib/locket",
+    "lib/lxml",
+    "lib/markupsafe",
+    "lib/matplotlib",
+    "lib/mpl_toolkits",
+    "lib/msgpack",
+    "lib/multiprocessing",
+    "lib/nbconvert",
+    "lib/nbformat",
+    "lib/netCDF4",
+    "lib/nose",
+    "lib/numexpr",
+    "lib/packaging",
+    "lib/pandas",
+    "lib/partd",
+    "lib/pbr",
+    "lib/pexpect",
+    "lib/PIL",
+    "lib/prompt_toolkit",
+    "lib/psutil",
+    "lib/ptyprocess",
+    "lib/py",
+    "lib/pydoc_data",
+    "lib/pygments",
+    "lib/pytz",
+    "lib/requests",
+    "lib/scipy",
+    "lib/setuptools",
+    "lib/simplejson",
+    "lib/sortedcontainers",
+    "lib/sqlite3",
+    "lib/tables",
+    "lib/tblib",
+    "lib/testpath",
+    "lib/toolz",
+    "lib/tornado",
+    "lib/traitlets",
+    "lib/urllib3",
+    "lib/wcwidth",
+    "lib/webencodings",
+    "lib/xmlrpc",
+    "lib/yaml",
+    "lib/zict",
+    "lib/zmq",
+    "lib/coverage",
+    "lib/distutils",
+    "lib/mock",
+    "lib/PyQt5",
+    "lib/babel",
+    "lib/sphinx",
+    "platforms",
+    "imageformats",
+    "definitions",
+    "mpl-data",
+    "lib/PySide2/Qt/qml",
```

Comment:
this will only work on linux as it contains forward slashes. Windows won't work properly here. Use os.path.join instead of the slashes like so: 

```python 
os.path.join(os.path.join(os.path.join("lib", "PySide2"), "Qt"), "qml")
```

I'm not sure if there's a nicer way to do this, but it reads OK.